src/test/java/gov/nist/secauto/oscal/lib/profile/resolver/ProfileResolutionTests.java

```diff
@@ -182,7 +182,8 @@ void performTest(String profileName) throws IOException, SaxonApiException {
     StringWriter writer = new StringWriter();
     serializer.serialize(catalog, writer);
 
-    // OscalBindingContext.instance().newSerializer(Format.YAML, Catalog.class).serialize(catalog,
+    // OscalBindingContext.instance().newSerializer(Format.YAML,
```

Comment:
Apologies, I worked in this file and auto-formatting with project rules triggered this. Revert?

Reply:
Woops, disregard.